src/handlers/pull-request/createDevPoolPR.ts

```diff
@@ -0,0 +1,83 @@
+import { getBotContext, getLogger } from "../../bindings";
+import { Payload } from "../../types";
+
+export const createDevPoolPR = async () => {
+  const logger = getLogger();
+
+  const context = getBotContext();
+  let payload = context.payload as Payload;
+
+  let devPoolOwner = "ubiquity";
+  let devPoolRepo = "devpool";
+
+  if (payload.repositories_added?.length === 0) {
+    return;
+  }
+
+  let repository = payload.repositories_added![0];
+
+  logger.info(`New Install: ${repository.full_name}`);
+
+  const [owner, repo] = repository.full_name.split("/");
+
+  // create a new branch
+  const branchName = `add-${owner}-${repo}`;
+  const baseRef = "development";
+  const path = "projects.json";
+
+  const { data: branch } = await context.octokit.repos.getBranch({
+    owner: devPoolOwner,
+    repo: devPoolRepo,
+    branch: "development",
+  });
+
+  // Get the current projects json file
+  const { data: file } = await context.octokit.repos.getContent({
+    owner: devPoolOwner,
+    repo: devPoolRepo,
+    path,
+    ref: "development",
+  });
+
+  const contentFile = JSON.parse(JSON.stringify(file));
```

Comment:
Object.assign is probably better if you want to copy an object.

Reply:
I'm not copying, i'm just parsing here..

Reply:
Shouldn't `file` already be parsed? Why are you stringifying it.

Reply:
> Shouldn't `file` already be parsed? Why are you serializing it?

We need to parse it, but for the stringify part.. it was only to make it parsable, the object returned from the getContent wasn't allowing a direct parse cos it was more string-like

Reply:
Can you paste the exact value here? It doesn't make any sense to me because I never had to do this other than for attempting to copy an object.

Reply:
> Can you paste the exact value here? It doesn't make any sense to me because I never had to do this other than for attempting to copy an object.

I used object assign it was a better way, no errors and it works.. i've made a new commit